docs/dev-guide/testnet.md

```diff
@@ -4,7 +4,7 @@ title: Testnet Aggron
 ---
 
 We are going to deploy a PoA (Proof of Authority) algorithm to create blocks in the testnet. However, it will take some time to develop. Before that we will reset the testnet regularily whenver there's no new blocks in an hour. 
-The chain spec used to start Aggron has been published [here](https://gist.github.com/doitian/573513c345165c0fe4f3504ebc1c8f9f). Please refer to the [Chains](https://github.com/nervosnetwork/ckb/wiki/Chains) document for Aggron Testnet’s info.
+The chain spec used to start Aggron has been published [here](https://github.com/nervosnetwork/ckb/wiki/Chains). Please refer to the [Chains](https://github.com/nervosnetwork/ckb/wiki/Chains) document for Aggron Testnet’s info.
```

Comment:
Duplicated links, maybe need to rephrase here.

Reply:
got it. thx!